R/process_data.R

```diff
@@ -344,6 +344,8 @@ process_scenario_data <- function(data, start_year, end_year, sectors, technolog
 process_carbon_data <- function(data, start_year, end_year) {
   data_processed <- data %>%
     dplyr::filter(dplyr::between(.data$year, .env$start_year, .env$end_year)) %>%
+      dplyr::filter(.data$model == "MESSAGEix-GLOBIOM 1.0") %>%
+      dplyr::select(-c(scenario_geography))  %>%
```

Comment:
why are u deselecting scenario_geogrpahy? It seems there is not filtering by it, so why include it in the first place? 🤔